LICENSE.md

```diff
@@ -0,0 +1,23 @@
+# License Agreement
+
+Up to date information for the whole [SpiNNakerManchester Projects](https://github.com/SpiNNakerManchester) can be found [here](http://spinnakermanchester.github.io/common_pages/4.0.0/LicenseAgreement.html)
```

Comment:
This and subsequent URLs should probably point to 6.0.0 rather than 4.0.0.

Reply:
Ah copy paste error!  Fixed now.